txtdirect.go

```diff
@@ -217,8 +228,8 @@ func Redirect(w http.ResponseWriter, r *http.Request, c Config) error {
 
 		if path != "" {
 			zone, from, err := zoneFromPath(host, path, rec)
-			rec, err = getFinalRecord(zone, from)
-			if err != nil {
+			rec, err = getFinalRecord(zone, from, r.Context(), c)
+      if err != nil {
```

Comment:
Formatting seems broken here or is that just Github acting up.

Reply:
I think formatting got broken when I used GitHub's editor to fix the conflict. Will fix it in the next commit.